forgerock-auth/src/main/java/org/forgerock/android/auth/selfservice/Session.kt

```diff
@@ -38,8 +39,8 @@ internal fun ssoToken(): SSOToken {
  * @throws IllegalArgumentException if the URL is not set.
  */
 fun ServerConfig.am(): Uri.Builder {
-    url?.let {
-        return Uri.parse(url).buildUpon()
+    return url?.let {
```

Comment:
Do you think the URI handling could be more null-safe as following?

```
fun ServerConfig.am(): Uri.Builder = url?.toUri()?.buildUpon()
    ?: throw IllegalArgumentException("URL is not set")
```

Reply:
Updated.